nipy/algorithms/group/parcel_analysis.py

```diff
@@ -267,7 +268,8 @@ def __init__(self, con_imgs, parcel_img, parcel_info=None,
         # the input full-width-at-half-maximum parameter given in mm
         # to standard deviation in voxel units.
         orient = io_orientation(self.affine)[:, 0].astype(int)
-        voxsize = np.abs(self.affine[(orient, range(3))])
+        # `orient` is an array, so this slicing leads to advanced indexing.
```

Comment:
Added a comment.   I still don't understand the difference between the list and the slice here, but oh well.